python/lsst/cp/verify/repackStats.py

```diff
@@ -0,0 +1,448 @@
+    "CpVerifyRepackDarkTask",
+    "CpVerifyRepackFlatTask",
+    "CpVerifyRepackDefectTask",
+    "CpVerifyRepackPtcTask",
+    "CpVerifyRepackBfkTask",
+    "CpVerifyRepackCtiTask",
+]
+
+
+class CpVerifyRepackConnections(pipeBase.PipelineTaskConnections,
+                                dimensions={"instrument"},
+                                defaultTemplate={}):
+    detectorStats = cT.Input(
+        name="detectorStats",
+        doc="Input detector statistics.",
+        storageClass="StructuredDataDict",
+        dimensions={"instrument", "exposure", "detector"},
+        multiple=True,
+    )
+    exposureStats = cT.Input(
+        name="exposureStats",
+        doc="Input exposure statistics.",
+        storageClass="StructuredDataDict",
+        dimensions={"instrument", "exposure"},
+        multiple=True,
+    )
+    runStats = cT.Input(
+        name="runStats",
+        doc="Input Run statistics.",
+        storageClass="StructuredDataDict",
+        dimensions={"instrument"},
+        multiple=True,
+    )
+
+    outputCatalog = cT.Output(
+        name="cpvCatalog",
+        doc="Output merged catalog.",
+        storageClass="ArrowAstropy",
+        dimensions={"instrument"},
+    )
+
+
+class CpVerifyRepackConfig(pipeBase.PipelineTaskConfig,
+                           pipelineConnections=CpVerifyRepackConnections):
+
+    expectedDistributionLevels = pexConfig.ListField(
+        dtype=float,
+        doc="Percentile levels expected in the calibration header.",
+        default=[0, 5, 16, 50, 84, 95, 100],
+    )
+
+
+class CpVerifyRepackTask(pipeBase.PipelineTask):
+    """Repack cpVerify statistics for analysis_tools.
+    """
+    ConfigClass = CpVerifyRepackConfig
+    _DefaultName = "cpVerifyRepack"
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+
+        inputs["detectorDims"] = [exp.dataId.byName() for exp in inputRefs.detectorStats]
+        inputs["exposureDims"] = [exp.dataId.byName() for exp in inputRefs.exposureStats]
+
+        outputs = self.run(**inputs)
+        butlerQC.put(outputs, outputRefs)
+
+    def run(self, detectorStats, detectorDims, exposureStats, exposureDims, runStats):
+        """
+        """
+        rowList = self.repack(detectorStats, detectorDims, exposureStats, exposureDims, runStats)
+        catalog = Table(rowList)
+
+        return pipeBase.Struct(
+            outputCatalog=catalog,
+        )
+
+    def repackDetStats(self, detectorStats, detectorDims):
+        raise NotImplementedError("Repack needs to be defined by subclasses.")
+
+    def repackExpStats(self, exposureStats, exposureDims):
+        # for expStats, expDims in zip(exposureStats, exposureDims):
+        raise NotImplementedError("Repack needs to be defined by subclasses.")
+
+    def repackRunStats(self, runStats):
+        # for runStats in runStats:
+        raise NotImplementedError("Repack needs to be defined by subclasses.")
```

Comment:
Are repackExpStats and repackRunStats used anywhere? It looks like it is just repackDetStats.

Reply:
They are not, but I wanted to be able to access them in the future if they became useful.